paywall/unlock.js.webpack.config.js

```diff
@@ -0,0 +1,22 @@
+var path = require('path')
+const webpack = require('webpack')
+
+module.exports = env => {
+  return {
+    mode: 'production',
+    devtool: 'source-map',
+    entry: path.resolve(__dirname, 'src', 'unlock.js', 'index.js'),
+    output: {
+      path: path.resolve(__dirname, 'src', 'static'),
+      filename: 'unlock.1.0.min.js',
```

Comment:
Let's exatract the version number into a constant at the top of this file so we can easily change later on?

Reply:
oh, FYI, I was fully expecting that if we bump the version, we will generate entirely separate scripts, and continue to generate the older versions, since clients will be depending on them. We can talk about this if it doesn't match your assumptions